README.md

```diff
@@ -98,4 +98,4 @@ parsable format as if it were XML, without the need for markup. </p>
 <p>Software to support ixml will be made available at a later date, at https://github.com/invisibleXML/ixml</p>
 
 <p>The draft <a href="ixml-specification.html">Specification for Invisible
```

Comment:
How about
```
<p>The draft <a href="https://invisiblexml.org/ixml-specification.html">Specification for Invisible XML</a> is available.</p>
```
instead?

Reply:
The idea was to display the file from GitHub, not another file. Michael asked if it was possible.

Reply:
Yes, I think this does that, too